deployment.py

```diff
@@ -139,7 +131,31 @@ def __init__(self):
                     {
                         "flow": "sym",
                         "proto": "all",
-                        "groups": ("internal", "web")
+                        "groups": ("internal", "temp", "web")
+                    },
+                )
+            },
+
+            {
+                "name": "temp",
+                "rules": (
+                    {
+                        "flow": "in",
+                        "proto": "tcp",
+                        "port": [22, 80, 443, 8080],
+                        "cidr_ip": "0.0.0.0/0"
+                    },
+
+                    {
+                        "flow": "out",
+                        "proto": "all",
+                        "cidr_ip": "0.0.0.0/0"
+                    },
```

Comment:
@manthey you are right that there is room to further lock things down.  In fact, I don't even think we'd need to explicitly allow outbound traffic over ephemeral ports since those connections would be ESTABLISHED (in iptables parlance).

Still, I'd rather not go down that route unless we have to -- these changes are primarily for restricting inbound connections.  Doing the same for outbound connections would require its own round of development so that, for example, we don't accidentally tank apt-get or girder_io.